src/main/resources/liquibase/changelogs/v1_30_3/schema.yml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+    - changeSet:
+        id: 1.30.3
+        author: GraviteeSource Team
+        changes:
+          - createTable:
+              tableName: alert_events
+              columns:
+                - column: {name: id, type: nvarchar(64), constraints: { nullable: false } }
```

Comment:
Please add here the PK constraint `- column: {name: id, type: nvarchar(64), constraints: { nullable: false, primaryKey: true } }`